EpsilonLights/Src/Lights.c

```diff
@@ -27,8 +27,13 @@ void updateLightsTask(void const* arg)
         hazards = (lightsInputs >> HAZARDS_INPUT_INDEX) & 1;
 
         /* UPDATE HEADLIGHTS */
-
-        if (headlightsOff && headlightsLow + headlightsHigh)
+        if ((headlightsOff) && (headlightsLow || headlightsHigh))
+        {
+            // Error state, turn headlights off
+            HAL_GPIO_WritePin(HHIGH_GPIO_Port, HHIGH_Pin, LIGHT_OFF);
+            HAL_GPIO_WritePin(HLOW_GPIO_Port, HLOW_Pin, LIGHT_OFF);
+        }
+        else if (headlightsLow && headlightsHigh)
```

Comment:
For safety, if it gets into an error state, wouldn't it be better to leave the headlights on? If it is daylight and the headlights turn on it wastes some power but not that bad. If it is dark out and it enters error state it would be dangerous to turn off all lights.

Reply:
Fair enough!